wgpu-hal/src/gles/egl.rs

```diff
@@ -543,11 +569,21 @@ impl crate::Instance<super::Api> for Instance {
                 egl::SINGLE_BUFFER as usize
             },
         ];
-        if inner.version >= (1, 5) {
-            // Always enable sRGB in EGL 1.5
-            attributes.push(egl::GL_COLORSPACE as usize);
-            attributes.push(egl::GL_COLORSPACE_SRGB as usize);
+
+        match inner.srgb_kind {
+            SrgbFrameBufferKind::Core => {
+                attributes.push(egl::GL_COLORSPACE as usize);
+                attributes.push(egl::GL_COLORSPACE_SRGB as usize);
+            }
+            SrgbFrameBufferKind::Khr => {
+                attributes.push(EGL_GL_COLORSPACE_KHR as usize);
+                attributes.push(EGL_GL_COLORSPACE_SRGB_KHR as usize);
+            }
+            SrgbFrameBufferKind::None => {
+                // TODO: expose ot the user that srgb isn't supported
```

Comment:
do we need this TODO? let's just implement it here in the PR

Reply:
I looked but couldn't find where I would change the list of supported backbuffer formats, so if you point me in the right direction I can do it.

Reply:
Sorry, I should have just posted the link right away - https://github.com/gfx-rs/wgpu/blob/22a81de6a99b25223f2fd9d6233f10fa16908ab6/wgpu-hal/src/gles/adapter.rs#L465